src/components/call/AssignInstrumentsToCall.tsx

```diff
@@ -80,6 +80,9 @@ const AssignInstrumentsToCall: React.FC<AssignInstrumentsToCallProps> = ({
         options={{
           search: true,
           selection: true,
+          headerSelectionProps: {
+            inputProps: { 'aria-label': 'Checkbox Select All Rows' },
```

Comment:
I think in general it's best to describe what the input does, rather than what the input is. A screen reader should already know that the element is a checkbox, so we should just describe it as "Select All Rows"